CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add missing UI translation keys.
+
+### Removes
+
+* Remove `@apostrophecms/multisite-18n` since it was previously used for multisite translations before [`@apostrophecms/multisite-dashboard`](https://github.com/apostrophecms/multisite-dashboard) existed.
```

Comment:
`@apostrophecms/multisite` 3.x series is still very much supported so we can't rip this out.

Reply:
It would be a big bc break for existing clients if that broke.